docs/self-managed/setup/deploy/amazon/amazon-eks/eksctl.md

```diff
@@ -108,7 +134,7 @@ The variable `KMS_ARN` contains the required output. It should look something li
 
 For more information concerning the KMS encryption, refer to the [eksctl documentation](https://eksctl.io/usage/kms-encryption/).
 
-### eksctl cluster YAML
+### Create the cluster using eksctl
```

Comment:
wondering about the yaml below whether it would make sense to put into `camunda-tf-eks-module` as well and reference.